### ISSUE TITLE
About hyperparameters

### ISSUE THREAD

**Issue by penguinboy1108** [SrishtiGautam/ProtoVAE#1]
A I have recently been doing a study on the interpretability of sound recognition, and I chose to use your model with some modifications, considering that the original physical meaning could be preserved intact. I used it for training by first converting the audio into a Meier spectrogram and then from the (c, h, w) of the image to (c,t) (the spectrogram was downscaled because it was a single channel image). I designed the encoder and decoder layers with reference to the model identified by the SVHM dataset, but I don't know what the design basis is for the hyperparameters latent and num_prototypes, so I would like to ask you. Also, I ran 40 epochs with the original parameter settings, and although the loss of both the training and test sets are decreasing, the accuracy of the training and test sets are not improving, and the accuracy of the network without the decoder and the corresponding loss is quite different from that of the network.
